sdk/webpubsub/Azure.Messaging.WebPubSub/CHANGELOG.md

```diff
@@ -1,7 +1,14 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
+## 1.0.0-beta.2 (2021-07-16)
 
+### Features Added
+- Added support for [RequestOptions](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/samples/ProtocolMethods.md#using-requestoptions-to-customize-behavior) on protocol methods.
+- Added support for API management.
+- Simplified dependencies.
```

Comment:
What does this one mean?  Should we call it out more clearly?

Reply:
I removed a dependency on non-track 2 JWT library

Reply:
Should we remove the entry or can we make it more user-friendly?

Reply:
How about we just say `Removed dependency on System.IdentityModel.Tokens.Jwt.` with the name of the specific library on the off chance a customer randomly started using types in it too?  (I _think_ that's the dependency we removed based on the dependencies listed at https://www.nuget.org/packages/Azure.Messaging.WebPubSub?)